.github/workflows/deploy-dex-explorer.yml

```diff
@@ -51,9 +51,10 @@ jobs:
       - name: deploy veil
         run: >
           nix develop --command
-          kubectl -n testnet rollout restart deployment veil &&
+          kubectl -n testnet rollout restart deployment dex-explorer &&
           nix develop --command
-          kubectl -n testnet rollout status deployment veil
+          kubectl -n testnet rollout status deployment dex-explorer
+
```

Comment:
The `dex-explorer` name is correct for the current deployments; can always rename later on. I'll handle any string substitutions in the deploy workflows as necessary, in follow-ups.